ruamel_yaml/meta.yaml

```diff
@@ -2,17 +2,19 @@ package:
   name: ruamel_yaml
   version: 0.11.14
 
+source:
+  fn: ruamel_yaml-0.11.14.tar.gz
+  url: https://bitbucket.org/ruamel/yaml/get/0.11.14.tar.gz
+  sha256: b78d394e192b6d0911bd59f794ceea88143fac4e373b76ed7e48143a2a29b33c
```

Comment:
We prefer tarballs at conda-forge and this ordering. So figured we could match this up a bit more to our expectations if there is no strong preference.